pom.xml

```diff
@@ -56,22 +54,19 @@
       <version>2.9</version>
       <scope>test</scope>
     </dependency>
-
-    <!-- Pipeline steps used in testing -->
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-basic-steps</artifactId>
-      <version>2.3</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-durable-task-step</artifactId>
-      <version>2.4</version>
-      <scope>test</scope>
-    </dependency>
```

Comment:
Did these need to be removed? They only have `test` scope.

Reply:
Only `workflow-cps` and `workflow-job` are required for the current tests. Therefore I cleaned up the rest.

Reply:
Yeah, they just made it more convenient when running manual tests.